javascript/react-js/router.md

```diff
@@ -111,3 +120,10 @@ This section contains helpful links to other content. It isn't required, so cons
 
 - [This video by The Net Ninja](https://www.youtube.com/watch?v=QUz3k2O3ZJU&ab_channel=TheNetNinja) is an excellent review of what we learned here.
 - For some extra practice/review, check out [this article](https://css-tricks.com/learning-react-router/).
+
+### Knowledge Checks
+
+- <a class="knowledge-check-link" href="#react-router">How do you add page routing to a React project?</a>
```

Comment:
We don't want to just wholesale copy the Learning Outcomes for our Knowledge Checks, they should be difference.

Reply:
Thanks for the feedback, I appreciate the clarification. I'll make some changes and resubmit :+1: